<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>483</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <widget class="QWidget" name="tab_statement">
      <attribute name="title">
       <string>Задание условий</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_42">
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_pic">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../resource/pic.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <spacer name="horizontalSpacer_17">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>7</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <layout class="QGridLayout" name="gridLayout_21">
         <item row="4" column="1">
          <layout class="QGridLayout" name="gridLayout_20">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>eps = 10^</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_eps">
             <property name="minimum">
              <number>-7</number>
             </property>
             <property name="maximum">
              <number>-1</number>
             </property>
             <property name="value">
              <number>-5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QGridLayout" name="gridLayout_40">
           <item row="0" column="0">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Шагов по x</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_nx">
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="gridLayout_39">
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_max_y">
             <property name="minimum">
              <double>-100.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="value">
              <double>1.570000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>l_y</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_36">
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_max_x">
             <property name="minimum">
              <double>-100.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>l_x</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <layout class="QGridLayout" name="gridLayout_41">
           <item row="0" column="0">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Шагов по y</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_ny">
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" rowspan="5">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>c</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_c">
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>by</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_by"/>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>bx</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_bx"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="3">
        <layout class="QGridLayout" name="gridLayout_31">
         <item row="3" column="1">
          <layout class="QGridLayout" name="gridLayout_26">
           <item row="0" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>phi_3</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_phi_3">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <layout class="QGridLayout" name="gridLayout_30">
           <item row="0" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>phi_4</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_phi_4">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_32">
           <item row="0" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>f</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_f">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QGridLayout" name="gridLayout_24">
           <item row="0" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>phi_2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_phi_2">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="gridLayout_19">
           <item row="0" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>phi_1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_phi_1">
             <property name="text">
              <string>cos(y)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="1">
          <layout class="QGridLayout" name="gridLayout_35">
           <item row="0" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>U(x, y)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_analytic">
             <property name="text">
              <string>x * cos(y)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" rowspan="6">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <layout class="QGridLayout" name="gridLayout_34">
         <item row="4" column="1">
          <layout class="QGridLayout" name="gridLayout_33">
           <item row="0" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>relax coefficient</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_relax">
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1.990000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.700000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_libman">
           <property name="text">
            <string>метод Либмана</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="checkBox_sor">
           <property name="text">
            <string>метод простых итераций 
с верхней релаксацией</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox_analytic">
           <property name="text">
            <string>Аналитическое решение</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBox_seidel">
           <property name="text">
            <string>метод Зейделя</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="5">
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_18">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="1">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>alpha_1</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_alpha_1">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>beta_1</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_beta_1">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>beta_2</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_beta_2">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="value">
                <double>-1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="1">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>alpha_2</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_alpha_2">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_16">
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>beta_3</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_beta_3">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="1">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>alpha_3</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_alpha_3">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QGridLayout" name="gridLayout_17">
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_beta_4">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>beta_4</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_13">
             <item row="0" column="1">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>alpha_4</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_alpha_4">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1" rowspan="4">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>label_pic</zorder>
      <zorder>horizontalSpacer_17</zorder>
      <zorder>label_3</zorder>
      <zorder>spinBox_eps</zorder>
      <zorder></zorder>
      <zorder></zorder>
      <zorder></zorder>
      <zorder></zorder>
      <zorder></zorder>
      <zorder>line_4</zorder>
     </widget>
     <widget class="QWidget" name="tab_solution">
      <attribute name="title">
       <string>Решение</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_38">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_37">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_28">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_27">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_22">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Шаг по x</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBox_x"/>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_sol_y"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_25">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_23">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Шаг по y</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBox_y"/>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_sol_x"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_29">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_err_x"/>
           </item>
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_err_y"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
